main.js

```diff
@@ -0,0 +1,11 @@
+var $start = document.getElementById('start')
+$start.addEventListener('click', function() {
+  var startWatch = setInterval(addSecond, 1000)
+})
+
+function addSecond() {
+  var $elapsed = document.getElementById('elapsed')
+  var elapsedSeconds = parseInt($elapsed.textContent, 10)
```

Comment:
Create an object that represents the timer instead of storing the timer in the DOM. 
You can use the timer object when updating the DOM, but the truth about the timers state should be outside of the DOM.